### DISCUSSION TITLE
Some Spirit Bears

### DISCUSSION THREAD

**Discussion by gateway** [rkhamilton/vqgan-clip-generator#37] [Share generated art]
Just some results im seeing iterating though some art styles and materials + a base text.  Also I used a nice reference image from one of my good artist friends.. 

Iterations where locked around 50.. I ended up with 40 images and choose about 20 nice ones.. also note these are 3k x 3k each so I had to down res them, but they look gorgeous in full resolution.

![image](https://user-images.githubusercontent.com/519563/138990432-40fb3c50-2970-41db-9c15-671d50ed5829.png)

**Comment by gateway** [comment]
this is my fav 
![image](https://user-images.githubusercontent.com/519563/138990805-a8e7129b-5b93-4e10-91d1-f924d77a672a.png)

**Reply by rkhamilton** [reply]
That’s great! Thanks for sharing it.

**Reply by gateway** [reply]
If i wanted to do a zoom video, just in.. the rotation stuff seems to always make people a bit sick or just feel off kilter.. what would you recommend for 10-20 seconds.. fps 15, encode, 30 or 60, Rife, x.. blending settings.. ?

**Reply by rkhamilton** [reply]
I don’t think I’m a master of how to best use the settings, so take this with a big grain of salt:
I’d save every 15, encode 15 fps, then RIFE -epx=2 (takes it to 60fps with interpolation). Z_smoother on, alpha=0.9. If you get a lot of flicker, alpha to 0.7.

**Reply by gateway** [reply]
thanks I just want to get a base to play with then tweak from there.
